hera_sim/interpolators.py

```diff
@@ -4,7 +4,26 @@
 
 import numpy as np
 from cached_property import cached_property
-from scipy.interpolate import RectBivariateSpline
+from scipy.interpolate import RectBivariateSpline, interp1d
+from hera_sim.data import DATA_PATH
+from os import path
+
+INTERP_OBJECTS = {"1d": ("beam", "bandpass"),
+                  "2d": ("Tsky_mdl", ) }
+
+def _check_path(datafile):
+    # if the datafile is not an absolute path, assume it's in the data folder
+    if datafile[0] != "/":
```

Comment:
I believe there's an `os.path` function for checking if a path is absolute (in an OS-independent way)